File: android/app/src/main/java/info/bagen/dwebbrowser/microService/user/CotJMM.kt
package info.bagen.dwebbrowser.microService.user

import info.bagen.dwebbrowser.microService.sys.jmm.JmmMetadata
import info.bagen.dwebbrowser.microService.sys.jmm.JmmNMM
import info.bagen.dwebbrowser.microService.sys.jmm.JsMicroModule

class CotJMM : JsMicroModule(
    JmmMetadata(
        id = "cot.bfs.dweb",
        server = JmmMetadata.MainServer(
            root = "/sys",
            entry = "/bundle/public.service.worker.js"
        ),
        icon = "https://www.bfmeta.info/imgs/logo3.webp",
        title = "Cot"
    )
) {
    init {
        // TODO 测试打开的需要把metadata添加到 jmm app
        JmmNMM.getAndUpdateJmmNmmApps()[mmid] = this
    }
}

File: next/dweb-browser/src/Platforms/iOS/MicroModule/User/CotJMM.cs
using DwebBrowser.MicroService.Sys.Jmm;

namespace DwebBrowser.MicroService.Sys.User;

public class CotJMM : JsMicroModule
{
    public CotJMM() : base(new JmmMetadata(
        "cot.bfs.dweb",
        new JmmMetadata.MainServer() { Root = "file:///jmm", Entry = "/cot.worker.js" }))
    {
        // TODO 测试打开的需要把metadata添加到 jmm app
        JmmNMM.GetAndUpdateJmmNmmApps().Add(Mmid, this);
    }
}

